### DISCUSSION TITLE
Downgrading to the last version of GTA V

### DISCUSSION THREAD

**Discussion by OXP3573** [scripthookvdotnet/scripthookvdotnet#1458] [General]
Despite the Nightly Builds the script that I want to use is broken and won't load properly, I will have to revert to the last update and use the stable version to run it.

Does anyone know where I can find the old update files easily or whether there's a tool to downgrade to previous versions that pulls all the files automatically? I've had a look around but can't find anything. 

Thanks in advance.

**Comment by nomakewan** [comment]
If you bought the game on Steam, there is a way to downgrade by grabbing the depot files from the version you're looking for.

[Downgrading legitimately-purchased Steam copy of GTAV](https://www.reddit.com/r/Gta5Modding/comments/1c5klwu/gta_5_downgrade_to_1030950_very_easy)

This post refers to downgrading to 3095, but if you want to downgrade to 3179 instead, just use manifest ID [3235923229384003759](https://steamdb.info/patchnotes/14026221). In case it isn't clear the manifest ID is that long number at the end of the console command.

If you did not purchase on Steam and did not back up your game files before the June 25 update, you are screwed as far as legal methods which can be discussed here on Github are concerned.
